app/controllers/user.js

```diff
@@ -0,0 +1,174 @@
+const jwt = require('jsonwebtoken');
+const model = require('../models');
+
+const secret = process.env.SECRET_TOKEN || 'secret';
+
+// To create text-friendly user details
+const formattedUser = (user) => {
+  const userDetails = {
+    id: user.id,
+    userName: user.userName,
+    firstName: user.firstName,
+    lastName: user.lastName,
+    email: user.email,
+    RoleId: user.RoleId,
+    createdAt: user.createdAt,
+    updatedAt: user.updatedAt
+  };
+  return userDetails;
+};
+
+/**
+ * Class UsersController
+ * To handle routing logic for documents route
+ */
+class UsersController {
+
+  /**
+   * Method getUsers to obtain all users
+   * @param {object} request - request object
+   * @param {object} response - response object
+   * @returns {Object} response object
+   */
+  static getUsers(request, response) {
+    model.User.findAll({
+      attributes: [
+        'id',
+        'userName',
+        'firstName',
+        'lastName',
+        'email',
+        'RoleId',
+        'createdAt',
+        'updatedAt'
+      ]
+    }).then((users) => {
+      return response.status(202)
```

Comment:
This request has been processed at this point so `200 - OK` status is appropriate.